### PR TITLE
Create info.ini

### PR DESCRIPTION
**Pull request by gustoase**

### CODE REVIEW

File: drozdov-ai-phone-call/info.ini [hunk 1]
```diff
@@ -0,0 +1,5 @@
+author = "Alexandr Drozdov"
```
Comment (megawac) [inline drozdov-ai-phone-call/info.ini]: Place this under the `files/` folder

Comment (gustoase) [inline drozdov-ai-phone-call/info.ini]: hi) I added all the files

**Comment by jimaek** [comment]
They are still not under /files

**Comment by jimaek** [comment]
Also your project lack any documentation, we require that

**Pull request closed without merging**